docs/modules/user-guide/partials/con_devfile.adoc

```diff
@@ -1,42 +1,46 @@
 [id="con_devfile_{context}"]
-= Devfile
+= About devfiles
 
 [role="_abstract"]
 
-Devfiles are YAML files that describe and define developer workspaces or cloud-native development environments. Devfiles also describe best practices for end-to-end application development and customize or extend the application development to fit a team's current development practices. Instead of forcing a new workflow or tools, devfiles provide instructions for automating manual processes.
+.What a devfile does for you
 
-Development tools consume devfiles and transform the definition into a cloud workspace composed of multiple containers and cloud-native resources. Also, users can create any number of identical workspaces from the same devfile. This way, different teams can use and share workspaces in various ways, including a devfile registry or together with the source code of a project.
+Devfiles are YAML files, and they simplify your development process. Devfiles capture instructions on how to set up your development environment and provide these instructions to your tools and IDE. The tools and IDE understand the instructions and support devfiles in order to take your development project from source code to a running application. By using devfiles, you can guarantee consistent patterns of behavior for applications written in different languages and using different tools. Devfiles also provide automation steps to your tools and IDE so that they can automate your current, manual processes. Using the recommended best practices from the devfile, the tools and IDE can: take in the repository hosting your application source code, build your code, run your application on your local container, and then deploy your application to cloud-native containers.
```

Comment:
```suggestion
You can use devfiles to automate and simplify your development process. With devfiles, you record the instructions for configuring and running your software build environment as a YAML-formatted text file. Then, optionally, you can distribute those instructions to other software build tools and IDEs that support devfiles. Finally, you run the devfile instructions to configure and build a running application from a development project. You can also use devfiles to provide consistent software build processes across different languages and tools. 
```

Reply:
It might be accurate or inaccurate--so look at this suggestion critically--but this is what I think the original was saying.

Reply:
Rolfe, this is a great re-write, thank you for taking the time to make these changes :) I'm definitely going to commit a lot of what you have written here. However, before I commit all of it, I would like a SME to confirm the accuracy of some of your rewrites (it's cool to see how you interrupted it, so I know what my content is conveying to users!) 

@elsony or @maysunfaisal do either of you (or any other SME reviewer) agree with the accuracy of the above changes? Specifically, the following lines:
```
With devfiles, you record the instructions for configuring and running your software build environment as a YAML-formatted text file. Then, optionally, you can distribute those instructions to other software build tools and IDEs that support devfiles. Finally, to automatically configure and build a running application from a development project, you run the devfile instructions.
```

Reply:
it sounds very abstract to me IMO

Reply:
Thanks, Maysun, in which case, I'll commit the changes.